src/client.c

```diff
@@ -394,7 +398,9 @@ bool handle_auth_query_response(PgSocket *client, PktHdr *pkt) {
 		}
 		break;
 	case 'D':	/* DataRow */
-		memset(&user, 0, sizeof(user));
+		memset(&real_user, 0, sizeof(real_user));
```

Comment:
Code cleanup to use two variables (simpler) rather than partial fields of `PgUser*` struct.